components/resources/quizzes/shared/types.ts

```diff
@@ -0,0 +1,18 @@
+export type QuizQuestion = {
```

Comment:
thank you for keeping the types organized so nicely! your attention to detail is incredibly valuable!